packages/client/src/components/global/sticky-container.vue

```diff
@@ -1,71 +1,63 @@
 <template>
 <div ref="rootEl">
 	<slot name="header"></slot>
-	<div ref="bodyEl">
+	<div ref="bodyEl" :data-sticky-container-header-height="headerHeight">
 		<slot></slot>
 	</div>
 </div>
 </template>
 
-<script lang="ts">
-import { defineComponent, onMounted, onUnmounted, ref } from 'vue';
+<script lang="ts" setup>
+import { onMounted, onUnmounted } from 'vue';
```

Comment:
CONTRIBUTING.md says:

> - **When creating a new component, please use the Composition API (with [setup sugar](https://v3.vuejs.org/api/sfc-script-setup.html) and [ref sugar](https://github.com/vuejs/rfcs/discussions/369)) instead of the Options API.**
> 	- Some of the existing components are implemented in the Options API, but it is an old implementation. Refactors that migrate those components to the Composition API are also welcome.
> 
> https://github.com/misskey-dev/misskey/blob/a29ff7b1fa45e01bc7c097e8d770a4c4eedf2a8f/CONTRIBUTING.md#vue

According to the above, ref sugar (e.g. $ref, $computed) is recommended. However, seeing that @syuilo has merged, it appears that it is acceptable not to use ref sugar.

Reply:
_Whoops._

To chime in on this, while I read the contribution guide, I skipped over the ref sugar part apparently, otherwise I would have used that.

I based how I wrote my refactorings on other components that already used `setup` before, which also used the imported `ref` instead of the sugar `$ref` or `$computed`. (ex. https://github.com/misskey-dev/misskey/blob/develop/packages/client/src/pages/favorites.vue)

We should probably get a decision down on this, for further refactoring, I'm fine using both.